lib/repository.py

```diff
@@ -56,6 +56,19 @@ def get_git_repository(remote_repo_url, parent_dir_path):
                                         proxy=CONF.get('http_proxy'))
 
 
+def get_svn_repository(remote_repo_url, repo_path):
+    """
+    Get a local subversion repository located in a directory.
+    If it does not exist, check it out from the specified URL.
+    """
+    if os.path.exists(repo_path):
+        return SvnRepository(remote_repo_url, repo_path)
+    else:
+        # TODO: setup HTTP proxy by editing ~/.subversion/servers
```

Comment:
There's no need to edit the servers file, we can set a command line option: http://stackoverflow.com/a/28503928